content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,187 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+This work follows the derivation in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+
+
+
+Introduction
+------------
+
+ .. figure:: images/MT_N_layered_Earth-1.hires.png
+    :align: center
+    :scale: 20% 
+    :name: MTlayeredEarth
+
+    Earth Model Configuration
+
+The problem setup is shown in the figure below (See :numref:`MTlayeredEarth`), where we have
+
+| -a layered Earth, each layer with its own physical properties \\(\\sigma, \\varepsilon, \\mu \\) 
+
+| -a plane wave coming from air, perpendicular to the Earth's surface, traveling along the axis Z (oriented downward), with an electric field \\(\\mathbf{E_x}\\) and a magnetic field \\(\\mathbf{H_y}\\)
+
+| - the origin of coordinate system coincides with the Earth's surface
+
+
+
+
+
+
+Governing Equations
+-------------------
+
+The governing equation for DC resistivity problem can be obtained from
+:ref:`Maxwell's equations <maxwell1_fundamentals_index>`. We start by
+considering the zero-frequency case, in which case, Maxwell's equations are
+
+.. math::
+	\nabla \times \mathbf{E_x} = - i \omega \mu \mathbf{H_y}
+	:label: Faraday
+	
+.. math::
+	\nabla \times \mathbf{H_y} = (\sigma + i \omega \varepsilon) \mathbf{E_x}
+    :label: Ampere
+
+Knowing that the divergent of **E** and **H** are equal to 0 here (no free charge)
+according to :eq:`gauss_electric_frequency` and :eq: `gauss_magnetic_frequency`, we can combine the equations to write the Helmhotz (wave propagation) equation for both **E** and **B** field:
+
+.. math::
+    \nabla ^2  \mathbf{E_x} + k^2 \mathbf{E_x} = 0
+    :label: E_wave_propagation_equation
+
+.. math::
+    \nabla ^2 \mathbf{H_y} + k^2 \mathbf{H_y} = 0
+    :label: H_wave_propagation_equation
+
+with k the wavenumber:
+
+.. math::
+    k = \sqrt{\omega ^2 \mu \epsilon - i \omega \mu \sigma }
+    :label: kwavenumber
+
+
+We usually assume that the displacement current is negligible, which means \\(\\sigma >> \\omega \\varepsilon\\). In this case 
+
+.. math::
+    k \simeq (1-j) \sqrt{ \frac{\omega \mu \sigma}{2} }
+    :label: kwavenumber_steadystate
+
+Taking the problem from the point of view of the electric field, we know the :ref:E_wave_propagation_equation has a solution in the form of:
+
+.. math::
+    E_x (z) = U e^{i k z} + D e^{-i k z}
```

Comment:
The notation here has changed a bit (before E_x was bf and now it isn't), so a statement clarifying that ``:math:`\mathbf{E_x} = E_x \mathbf{\hat{x}}` `` would be helpful